ROADMAP-TO-1.0.0.md

```diff
@@ -156,16 +109,15 @@ These are some libraries that must exist to make developers' lives easy. It is c
   - [x] ipfs api bindings in Java
   - [x] ipfs api bindings in Python
   - [x] ipfs api bindings in PHP
-  - [ ] ipfs api bindings in Ruby (in progress)
-  - [ ] ipfs api bindings in Scala (in progress)
-  - [ ] ipfs api bindings in Rust (in progress)
-  - [ ] ipfs api bindings in Haskell
-  - [ ] ipfs api bindings in C/C++
+  - [x] ipfs api bindings in Ruby
+  - [x] ipfs api bindings in Scala
+  - [x] ipfs api bindings in Rust
+  - [x] ipfs api bindings in Haskell
+  - [x] ipfs api bindings in C/C++
```

Comment:
Someone should review each of these API bindings for whether they're complete, tested, and meet a certain quality